docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -44,10 +44,10 @@ Change directories to *FSNetCore* and start adding projects to the solution fold
 Use the `dotnet new` command, create a Class Library project in the **src** folder named Library. 
 
 ```bash
-dotnet new classlib -lang F# -o src/Library 
+dotnet new lib -lang F# -o src/Library 
```

Comment:
Sounds good guys! I'll merge this PR then. Thanks for the clarification on how it works on F#. Always learning.
Should we update docs for [dotnet new](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new) then? We just mention class library there. If it's not called that in F#, i think we should it make clearer and perhaps mention the partial match and add the example of a partial match. Thoughts?